tslint.json

```diff
@@ -0,0 +1,12 @@
+{
+  "defaultSeverity": "error",
+  "extends": [
+    "tslint-eslint-rules",
+    "tslint-config-airbnb",
+    "tslint-react",
+    "tslint-config-prettier"
+  ],
+  "rules": {
+    "import-name": false
```

Comment:
I haven't yet figured out all the conversions from our eslint setup. We can adjust as we go.